misc/build_helpers/build-manylinux-wheels.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
+
+# This runs in the container to actually build the wheels.
+BUILDER="/io/misc/build_helpers/_build-wheels.sh"
+
+# This image can build x86_64 (64 bit) manylinux wheels.
+DOCKER_IMAGE="quay.io/pypa/manylinux1_x86_64"
```

Comment:
https://docs.docker.com/engine/security/trust/content_trust/ describes the system.  It seems to work with quay.io.  So we could embed the sha256 of the expect image into the script and, if we trust Docker's "content trust" system then that ensures we're using exactly the image we wanted to.

Getting the sha256 of the image we want to use is a little trickier.  quay.io builds the image so even if I audit the Dockerfile I can't really know the image quay.io is serving me _now_ actually agrees with that Dockerfile.  Maybe I can build the image myself, locally, and it will end up with the same sha256?  But I doubt it because timestamps and stuff.